app/scripts/views/sign_up.js

```diff
@@ -197,7 +198,23 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
     },
 
     suggestEmail: function () {
-      mailcheck(this.$el.find('.email'), this.metrics, this.translator, this.window.location.search);
+      var abChoose;
+      var abData = {
+        isMetrics: !! this.metrics._able,
```

Comment:
I'm gonna be super pedantic here, can you rename `isMetrics` to `isMetricsEnabled`? If I understand correctly, *only* users with metrics enabled can take part in the experiment, and of those users 50% have mailcheck enabled, and 50% do not.